scripts/build_test.py

```diff
@@ -201,43 +202,43 @@ def test_compare_file_count(self):
         # Reset EMPTY_DIRECTORY to clean state.
         build.safe_delete_directory_tree(EMPTY_DIR)
 
-    def test_verify_filepath_hash(self):
+    def test_verify_filepath_hash(self) -> None:
         """Test _verify_filepath_hash raises exception:
             1) When there is an empty hash dict.
             2) When a filename is expected to contain hash but does not.
             3) When there is a hash in filename that cannot be found in
                 hash dict.
         """
         # Final filepath example: base.240933e7564bd72a4dde42ee23260c5f.html.
-        file_hashes = {}
+        file_hashes: Dict[str, str] = {}
         base_filename = 'base.html'
-        with self.assertRaisesRegex(ValueError, 'Hash dict is empty'):
+        with self.assertRaisesRegex(ValueError, 'Hash dict is empty'):  # type: ignore[no-untyped-call]
             build._verify_filepath_hash(base_filename, file_hashes)  # pylint: disable=protected-access
 
         # Generate a random hash dict for base.html.
-        file_hashes = {base_filename: random.getrandbits(128)}
-        with self.assertRaisesRegex(
+        file_hashes = {base_filename: 'abcdfabcdfabcdf1ABCDFABCDFABCDF1'}
```

Comment:
Why do you change this?

Reply:
Here we are using `Dict[str, str]` type for `file_hashes` and for those integer values in the hash dict, mypy was giving errors like this
### scripts/build_test.py:220: error: Dict entry 0 has incompatible type "str": "int"; expected "str": "str"  [dict-item]
### scripts/build_test.py:235: error: Argument 2 to "_insert_hash" has incompatible type "int"; expected "str"  [arg-type]
So, to remove them I added string value.

Reply:
Can you use some random function here that will generate a string instead of int?

Reply:
I tried to add a function in utils to generate string. PTAL